src/applications/personalization/dashboard-2/components/NotInMPIError.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
+import Telephone, {
+  CONTACTS,
+} from '@department-of-veterans-affairs/component-library/Telephone';
+
+const NotInMPIError = () => {
+  const content = (
+    <>
+      <p>
+        We can’t give you access to VA.gov tools to manage your health and
+        benefits right now. We’ll need to match your information and verify your
+        identity first.
+      </p>
+      <p>
+        If you’d like to access these tools on VA.gov, please contact the VA
+        help desk at <Telephone contact={CONTACTS.VA_311} /> (TTY:{' '}
+        <Telephone contact={CONTACTS['711']} />) to verify and update your
+        records.
+      </p>
+    </>
+  );
+
+  return (
+    <AlertBox
```

Comment:
I think this could potentially throw an a11y error for non-linear heading progression if it's rendered with the Dashboard Header, since the default heading level is `h3`. You can pass `level=2` as a prop to generate an `h2` instead.

Reply:
Oh good call. Will add the axe check in Cypress that should reveal this a11y error and fix this component

Reply:
Interesting. `cy.axeCheck();` doesn't seem to care about this sort of violation.